plugins/workload-management/src/main/java/org/opensearch/plugin/wlm/rest/RestCreateWorkloadGroupAction.java

```diff
@@ -55,6 +62,9 @@ public List<Route> routes() {
 
     @Override
     protected RestChannelConsumer prepareRequest(RestRequest request, NodeClient client) throws IOException {
+        if (workloadManagementSettings.getWlmMode() == WlmMode.DISABLED) {
+            throw new IllegalStateException("Workload management mode is DISABLED. Cannot create workload group.");
```

Comment:
Lets make this message more actionable and user friendly. It is missing what needs to be changed and why it needs to be changed

To address `what` we can surface the setting name and for `why` we should clearly state that workload group can only be created when the feature is enabled.